data/hai missions.txt

```diff
@@ -1053,3 +1053,245 @@ mission "Hai Honeymoon"
+			`	Turns out the documents she is holding are the bank papers for the loan, somewhat similar to the one you took when you bought your first ship. You've come a long way since your days as a beginner, and know how hard it is to be starting out as a starship captain, you could probably convince someone here to offer a simple cargo job to this young Hai.`
+			choice
+				`	(Offer to help her find some jobs.)`
+				`	(Wish her luck and leave.)`
+					decline
+			`	Her ears twitch a bit, and her previously crestfallen eyes light up as she nearly jumps in joy at your offer. "Thank you so much captain! My name is Nanachi, I promise I won't cause any trouble at all, just a few quick jobs and I'm sure I'll be fine by myself!"`
+			`	"Captain <first> <last>. Can you show me your ship so I can look for jobs it's better suited for?"`
+			`	She hesitates a bit, but leads you to her ship: you were expecting a small Hai ship, but to your surprise she shows you a rather old-looking Heavy Shuttle docked near your own ship.`
+			`	"You took a loan to buy a human ship?" you ask.`
+			`	She looks at the ground, as if ashamed of it. "It'd take way too long for me to qualify for a loan for even an Aphid, but I recently met a human merchant who wanted to sell his old ship; it was difficult but I managed to convince him to sell it to me instead of going to the shipyard office, I paid him its full price, but even for this one I had to take out the biggest loan I could. Its cargo hold isn't very impressive but it's got more bunks than our smallest ships, if that helps."`
+			choice
+				`	(Help her check the job board.)`
+				`	(Try approaching some people here looking for transport or for a courier.)`
+			`	Turns out this isn't as easy as you thought it'd be, the spaceport is very empty, save for the occasional worker or drunken crewmember, and the desk workers by the job board refuse to even let you see the job list because Nanachi is with you, it seems her inability to get a job has given her a bad reputation here. Eventually, as you two begin scouting the edge of the docking area, you spot a troubled merchant near several crates. You approach him and he explains that his ship had some malfunctions and he cannot deliver this advanced surgical equipment to Cloudfire.`
+			`	"This is the worst! One jump away from the planet and the navigation system suddenly decides to go on strike! There's no way I can make it before the deadline!"`
+			`	"My friend Nanachi here could deliver it for you. We were heading there anyway, a few extra crates won't make a difference." you say, trying to sound as convincing as possible.`
+			`	"Look kid, Unfettered attacks make it so they need a lot of medical supplies in Cloudfire, and this here is the best medical equipment Hai space has to offer, these few crates could save many military lives and I don't think I should be trusting something like this to a beginner." he says.`
```

Comment:
How does he know Nanachi is a beginner?
 
This response seems a little overly hostile when he was just given the chance to get his goods where they need to go on time. He basically has three choices with four outcomes: he repairs his ship first and gets the goods there too late, he gives up and the goods never get there, he gives them to Nanachi and she gets blown up, or he gives them to Nanachi and they make it to where they need to go. The only outcome that has the supplies getting to Cloudfire on time is giving them to Nanachi.

Maybe change this dialog to have this merchant pondering those options, ultimately deciding to risk the supplies with Nanachi as that at least means they'll get there on time. And remember, the player is escorting Nanachi, so it isn't like you're telling this guy that a single heavy shuttle (which hasn't even been mentioned) is flying on its own into dangerous territory to deliver some goods.

If you take that change, most of the rest of this conversation would be thrown out, so I'm just going to skip over it.